lib/repl.js

```diff
@@ -844,10 +844,49 @@ REPLServer.prototype.createContext = function() {
   });
 
   addBuiltinLibsToObject(context);
+  previewResults.call(this);
```

Comment:
This shouldn't be necessary - can you instead add the function on the class but either prefix the function name with `_` or, better, put it behind a Symbol? Something like `kPreviewResults = new Symbol('Preview Results Fn')`, and then you can do `this[kPreviewResults]()`.